mypyc/ops.py

```diff
@@ -423,15 +423,22 @@ class BasicBlock:
     as the final op in a block. Manually inserting error checking ops
     would be boring and error-prone.
 
+    BasicBlock's have an error_handler attribute that determines where
```

Comment:
Grammat nit: use "BasicBlocks" or "Basic blocks"

Reply:
Hm. Really I want it to be `BasicBlock`s, I guess, but...